### ISSUE TITLE
documentation for S needs improving

### ISSUE THREAD

**Issue by stephens999** [willwerscheid/flashier#129]
I think it is not clear how S is used from the docs?

**Comment by pcarbo** [comment]
I thought S specifies the residual variance parameters s_ij?

**Comment by stephens999** [comment]
that is true if var_type = NULL, but other cases are more complex i believe.

we should maybe note under the documentation for param S that the behavior
depends on var_type and then improve the description under var_type? But I'm not sure...

@willwerscheid 

Matthew
